vector/src/main/java/im/vector/app/features/home/HomeDetailViewModel.kt

```diff
@@ -204,21 +206,25 @@ class HomeDetailViewModel @AssistedInject constructor(@Assisted initialState: Ho
                         }
                         is RoomGroupingMethod.BySpace       -> {
                             val activeSpaceRoomId = groupingMethod.spaceSummary?.roomId
-                            val dmInvites = session.getRoomSummaries(
-                                    roomSummaryQueryParams {
-                                        memberships = listOf(Membership.INVITE)
-                                        roomCategoryFilter = RoomCategoryFilter.ONLY_DM
-                                        activeSpaceFilter = activeSpaceRoomId?.let { ActiveSpaceFilter.ActiveSpace(it) } ?: ActiveSpaceFilter.None
-                                    }
-                            ).size
+                            var dmInvites = 0
+                            var roomsInvite = 0
+                            if (!autoAcceptInvites.hideInvites) {
```

Comment:
Could be nicer if we also had a method `autoAcceptInvites.showInvites()`

Reply:
Yes